packages/lib/src/components/P24/index.ts

```diff
@@ -0,0 +1,7 @@
+import IssuerListContainer from '../helpers/IssuerListContainer';
+
+class P24Element extends IssuerListContainer {
+    public static type = 'dotpay';
```

Comment:
Are we sure that tx variant is `dotpay` ? It seems that the ticket description mentions `onlineBanking_PL`